CMakeLists.txt

```
@@ -47,6 +47,15 @@ include(cmake/XLinkDependencies.cmake)
 
 # Link to libusb
 target_link_libraries(${TARGET_NAME} PRIVATE usb-1.0)
+if(WIN32 AND NOT MINGW)
+    # needs minor fixes so that target_link_options/STATIC_LIBRARY_OPTIONS
+    # only puts the options on the thing that "contains" XLink executable bytes
+    target_link_libraries(${TARGET_NAME} PRIVATE delayimp.lib Pathcch.lib)
+    # BUGBUG private fails...it does not put /delayload on tests or examples
+    # interface and public both cascade up and put it on things "higher" than xlink and depthai-core
+    # I think this is a bug in depthai-core cmake
+    target_link_options(${TARGET_NAME} PUBLIC "/DELAYLOAD:libusb-1.0$<$<CONFIG:Debug>:d>.dll")
```

Okay, it seems we have this as generator expression: https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html?highlight=link_only#genex:LINK_ONLY

I think this is actually the "solve it all" for the discussed issues at hand

```suggestion
    target_link_options(${TARGET_NAME} PUBLIC "$<LINK_ONLY:/DELAYLOAD:libusb-1.0$<$<CONFIG:Debug>:d>.dll>")
```